<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDefinitionDialog</class>
 <widget class="QDialog" name="SetDefinitionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>711</width>
     <height>581</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Unit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Board</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>LineNo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Color</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Max</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Comment</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>690</y>
     <width>131</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Unit count</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCountOfUnit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCountOfBoard">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>board count in unit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetCount">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Set count</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonTestLED">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lamp on board</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
